testing/src/test/java/org/motechproject/nms/testing/it/flw/FrontLineWorkerImportServiceBundleIT.java

```diff
@@ -0,0 +1,165 @@
+import org.ops4j.pax.exam.junit.PaxExam;
+import org.ops4j.pax.exam.spi.reactors.ExamReactorStrategy;
+import org.ops4j.pax.exam.spi.reactors.PerSuite;
+
+import javax.inject.Inject;
+import java.io.Reader;
+import java.io.StringReader;
+import java.util.Arrays;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.motechproject.nms.testing.it.utils.LocationDataUtils.createCircle;
+import static org.motechproject.nms.testing.it.utils.LocationDataUtils.createDistrict;
+import static org.motechproject.nms.testing.it.utils.LocationDataUtils.createLanguage;
+import static org.motechproject.nms.testing.it.utils.LocationDataUtils.createLanguageLocation;
+import static org.motechproject.nms.testing.it.utils.LocationDataUtils.createState;
+
+@RunWith(PaxExam.class)
+@ExamReactorStrategy(PerSuite.class)
+@ExamFactory(MotechNativeTestContainerFactory.class)
+public class FrontLineWorkerImportServiceBundleIT extends BasePaxIT {
+
+    @Inject
+    private NationalDefaultLanguageLocationDataService nationalDefaultLanguageLocationDataService;
+    @Inject
+    private LanguageDataService languageDataService;
+    @Inject
+    private LanguageLocationDataService languageLocationDataService;
+    @Inject
+    private StateDataService stateDataService;
+    @Inject
+    private DistrictDataService districtDataService;
+    @Inject
+    private TalukaDataService talukaDataService;
+    @Inject
+    private VillageDataService villageDataService;
+    @Inject
+    private HealthBlockDataService healthBlockDataService;
+    @Inject
+    private HealthFacilityTypeDataService healthFacilityTypeDataService;
+    @Inject
+    private HealthFacilityDataService healthFacilityDataService;
+    @Inject
+    private HealthSubFacilityDataService healthSubFacilityDataService;
+    @Inject
+    private CircleDataService circleDataService;
+    @Inject
+    private FrontLineWorkerDataService frontLineWorkerDataService;
+
+    @Inject
+    private FrontLineWorkerImportService frontLineWorkerImportService;
+
+    @Before
+    public void setUp() {
+        for (FrontLineWorker flw: frontLineWorkerDataService.retrieveAll()) {
+            flw.setStatus(FrontLineWorkerStatus.INVALID);
+            flw.setInvalidationDate(new DateTime().withDate(2011, 8, 1));
+
+            frontLineWorkerDataService.update(flw);
+        }
+        frontLineWorkerDataService.deleteAll();
+        nationalDefaultLanguageLocationDataService.deleteAll();
+        languageLocationDataService.deleteAll();
+        languageDataService.deleteAll();
+        healthSubFacilityDataService.deleteAll();
+        healthFacilityDataService.deleteAll();
+        healthFacilityTypeDataService.deleteAll();
+        healthBlockDataService.deleteAll();
+        villageDataService.deleteAll();
+        talukaDataService.deleteAll();
+        districtDataService.deleteAll();
+        stateDataService.deleteAll();
+        circleDataService.deleteAll();
+
```

Comment:
Instead of calling all of the deleteAll you can call TestingServiceImpl.clearDatabase()